stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/input/MessageInputViewStyle.kt

```diff
@@ -84,6 +92,12 @@ public data class MessageInputViewStyle(
     public val customCursorDrawable: Drawable?,
     public val attachmentMaxFileSize: Int,
     public val dividerBackground: Drawable,
+    public val pictureAttachmentIcon: Drawable,
```

Comment:
We don't need all these attributes directly in MessageInputView? Right? We can keep them in `val attachmentDialogStyle: AttachmentDialogStyle`. We have similar in `MessageListViewStyle::scrollButtonViewStyle`